main.py

```diff
@@ -21,6 +21,10 @@ def get(self):
 		template_vals = generate_base_template_vals(self)
 		template_vals['page'] = 'home'
 		
+		# Get the home page text.
+		home_content = PageContent.query(PageContent.page == 'home').get()
+		template_vals['home_html'] = home_content.html_text
```

Comment:
@derekw023: I think the bug you found should go away if this line is changed to
```template_vals['home_html'] = '' if not home_content else home_content.html_text```
but I will wait and test it after work before committing it.